service/prometheus/service.go

```diff
@@ -0,0 +1,81 @@
+package prometheus
+
+import (
+	"net/http"
+	"net/url"
+	"path"
+
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+)
+
+const (
+	prometheusReloadPath = "/-/reload"
+)
+
+type Prometheus interface {
```

Comment:
I feel like this is in the wrong place. Input required.

Reply:
We put these things into a `spec.go` usually.

Reply:
Vote for `prometheus.go`.

Reply:
Done.

Reply:
Sorry pawel, it went into `spec.go` :D